plugin-tests/src/test/scala/com/olegpy/bm4/TestNoMap.scala

```diff
@@ -74,26 +54,4 @@ class TestNoMap extends FreeSpec {
       new MapCheck(()).flatMap(y => new MapCheck(()).map(x => x))
     }
   }
-
-  "works in generic context with extension methods of cats" in {
-    import cats.syntax.all._
-    def sillyTest[F[_]: cats.FlatMap](fa: F[Int], fb: F[Int]) =
-      for {
-        _ <- fa
-        b <- fb
-      } yield b
-
-    sillyTest(new MapCheck(11), new MapCheck(42))
-  }
-
-  "works in generic context with extension methods of scalaz 7" in {
-    import scalaz._, Scalaz._
-    def sillyTest[F[_]: Bind](fa: F[Int], fb: F[Int]) =
-      for {
-        _ <- fa
-        b <- fb
-      } yield b
-
-    sillyTest(new MapCheck(11), new MapCheck(42))
```

Comment:
Was this little test fully removed? It doesn't add much, but I'd rather keep it in case of whatever changes I do later on :)

Reply:
I made a decision some time ago not only to exit that community, but to stop writing code for it.  I have seen my friends mocked by name for their personal appearance and repeated memes that target the marginalized.  My kids are learning to code, and my son with autism would have to learn why they go "reee" for the lulz, and my biracial daughter the subtle nuances between their functional Pepes and white nationalist Pepes.  The code of conduct focuses on technical excellence, but equal excellence is found in places that don't protect bullies.  I'm not going to drive traffic to a place my friends and family feel unwelcome.

If this test is important to you, it's an easy copy and paste to restore it.

Reply:
Ok, I'll restore it myself.